src/components/contextpopup/ContextPopupDirective.js

```diff
@@ -52,7 +54,7 @@ goog.require('ga_window_service');
             var formatCoordinates = function(coord, prec, ignoreThousand) {
```

Comment:
you can remove this function because she does the same as `gaMeasure.formatCoordinates(coord, prec)` . Replace the use of it by `gaMeasure.formatCoordinates(coord, prec)` directly.